host-controller/src/main/java/org/jboss/as/domain/controller/resources/ProfileResourceDefinition.java

```diff
@@ -98,6 +98,6 @@ public void registerAttributes(ManagementResourceRegistration resourceRegistrati
     }
 
     public static OperationStepHandler createIncludesValidationHandler() {
-        return new DomainIncludesValidationWriteAttributeHandler(INCLUDES);
+        return new DomainIncludesValidationWriteAttributeHandler();
```

Comment:
We could use a `DomainIncludesValidationWriteAttributeHandler.INSTANCE ` since it is used in several places, I leave it up to you @pferraro